src/App.css

```diff
@@ -1,41 +1,76 @@
-#root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-  text-align: center;
-}
 
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
+
+.header {
+  width: 100%;
+  height: 100px;
+  background-color: teal;
+  text-align: center;
+  position:sticky
 }
-.logo:hover {
-  filter: drop-shadow(0 0 2em #646cffaa);
+.btn {
+  padding: 5px;
+  background: transparent;
+  border: none;
+  color: white;
+  font-family: cursive;
+  font-size: 25px;
+  cursor: pointer;
+  margin-top: 1.4rem;
+  position: relative;
+  overflow: hidden;
 }
```

Comment:
Component specific styles should be written in a separate file alongside its respective component. For eg:
src/components/Navbar should contain, Navbar.tsx, Navbar.css and index.ts